ui/nuxeo-data-table/iron-data-table.js

```diff
@@ -159,6 +159,10 @@ import '../nuxeo-button-styles.js';
             color: var(--paper-input-container-invalid-color, #de350b);
           }
 
+          :host([settings-enabled]) ::slotted(nuxeo-data-table-row:not([header])) {
+            padding-inline-end: 21px;
```

Comment:
why 21 px?

Reply:
Sorry, it should have been 1.5em, because of this: https://github.com/nuxeo/nuxeo-elements/blob/maintenance-3.0.x/ui/nuxeo-data-table/data-table-settings.js#L34-L35

Reply:
Or should we just put 21px everywhere?

Reply:
I would be in favor to change it to relative units, such as `rem`. My question about the `21px` was more about the value itself. Ideally we would be using multiples of four, according to our design guidelines, whenever is possible.